audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAuditWriter.java

```diff
@@ -281,12 +287,15 @@ public void scheduledFlush() {
     }
 
     /**
-     * Scheduled cleanup of audit data in database.
+     * Scheduled cleanup of audit data in the database.
      */
-    @Scheduled(fixedDelayString = "${audit.cleanup.delay.fixed:3600000}", initialDelayString = "${powerauth.audit.cleanup.delay.initial:1000}")
+    @Scheduled(cron = "${audit.cleanup.cron:0 0 0/1 * * *}")
+    @SchedulerLock(name = "audit.cleanup", lockAtLeastFor = "${audit.cleanup.lockAtLeastFor:5s}", lockAtMostFor = "${audit.cleanup.lockAtMostFor:50m}")
```

Comment:
I am not sure `50m` is a reasonable default for `lockAtMostFor` because it will leave just 10 minutes for scheduling all other jobs, supposing there is only 1 scheduling thread. Perhaps something more balanced like `30m` is better.

Reply:
Ok, let's change it to 30m.